pkg/webhook/authority/authority.go

```diff
@@ -310,7 +312,7 @@ var serialNumberLimit = new(big.Int).Lsh(big.NewInt(1), 128)
 // If the provided Secret is nil, a new secret resource will be Created.
 // Otherwise, the provided resource will be modified and Updated.
 func (d *DynamicAuthority) regenerateCA(ctx context.Context, s *corev1.Secret) error {
-	d.Log.Info("Generating new root CA")
+	d.Log.V(logf.DebugLevel).Info("Generating new root CA")
```

Comment:
These are logged at Debug where as the corresponding messages above, which end `Regenerating...` are at InfoLevel.
So if you're filtering logging at InfoLevel, you'll never see the continuation of the `...` messages above.
Long story short. I think we should remove the `...` in messages above :-)